packages/itwinui-css/backstop/tests/iModel-loader.html

```diff
@@ -49,7 +53,7 @@
           <g>
             <polygon
               data-id="outer3"
-              fill="#62bb47"
+              fill="var(--iui-color-icon-positive)"
```

Comment:
@gretanausedaite I've updated this to use CSS vars.  I recall we had some color assignment issues in the past, is there any issue using CSS vars here instead of HEX?

Reply:
There is no issue in this example. We have this loader in our illustrations package (and we recommend developers use it from there, if I remember correctly). The problem in illustrations package - we can't use CSS vars without variables package. So we should have some kind of hardcoded value there.

We can leave this issue for after new releases. As example code, this looks good to me.

Reply:
I just confirmed that something like this works:
```html
fill="var(--iui-color-icon-positive, hsl(118 74% 35%))"
```
If iTwinUI-Variables is present, it uses the CSS variable value.  If iTwinUI-Variables i not present, it uses the hardcoded value to the right (which is the same as light theme's `--iui-color-icon-positive`).